src/discord_mcp/core/plugins/manager.py

```diff
@@ -13,16 +15,35 @@
     TokenBucketRateLimiter,
     get_bucket_key,
 )
+from discord_mcp.core.server.middleware import MiddlewareContext
 from discord_mcp.core.server.shared.manifests import BaseManifest, PromptManifest, ResourceManifest, ToolManifest
 from discord_mcp.utils.enums import RateLimitType
 
 if t.TYPE_CHECKING:
     from discord_mcp.core.server.shared.context import DiscordMCPContext
 
-__all__: tuple[str, ...] = ("DiscordMCPPluginManager",)
+__all__: tuple[str, ...] = (
+    "DiscordMCPPluginManager",
+    "ManifestT",
+    "CoroFuncT",
+    "PredicateRequestT",
+    "PredicateT",
+    "Check",
+)
```

Comment:
```suggestion
__all__: tuple[str, ...] = (
    "DiscordMCPPluginManager",
    "Check",
)
```
`__all__` determines the members to be imported when using the star `*` import, type variabile and other typing only members are generally not included